diff --git a/pkg/osquery/table/platform_tables_linux.go b/pkg/osquery/table/platform_tables_linux.go
index ac9aad1..3aba050 100644
--- a/pkg/osquery/table/platform_tables_linux.go
+++ b/pkg/osquery/table/platform_tables_linux.go
@@ -24,7 +24,7 @@ func platformTables(client *osquery.ExtensionManagerClient, logger log.Logger, c
 		fscrypt_info.TablePlugin(logger),
 		dataflattentable.TablePluginExec(client, logger,
 			"kolide_nmcli_wifi", dataflattentable.KeyValueType,
-			[]string{"/usr/bin/nmcli", "--mode=multiline", "--fields=all", "device", "wifi", "list"},
+			[]string{"nmcli", "--mode=multiline", "--fields=all", "device", "wifi", "list"},
 			dataflattentable.WithKVSeparator(":")),
 		dataflattentable.TablePluginExec(client, logger, "kolide_lsblk", dataflattentable.JsonType,
 			[]string{"lsblk", "-J"},
diff --git a/pkg/osquery/tables/cryptsetup/table.go b/pkg/osquery/tables/cryptsetup/table.go
index 426a2ee..fff00ab 100644
--- a/pkg/osquery/tables/cryptsetup/table.go
+++ b/pkg/osquery/tables/cryptsetup/table.go
@@ -17,6 +17,7 @@ import (
 var cryptsetupPaths = []string{
 	"/usr/sbin/cryptsetup",
 	"/sbin/cryptsetup",
+	"cryptsetup",
 }
 
 const allowedNameCharacters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-/_"
diff --git a/pkg/osquery/tables/gsettings/gsettings.go b/pkg/osquery/tables/gsettings/gsettings.go
index 0237106..c264559 100644
--- a/pkg/osquery/tables/gsettings/gsettings.go
+++ b/pkg/osquery/tables/gsettings/gsettings.go
@@ -26,7 +26,7 @@ import (
 	"github.com/pkg/errors"
 )
 
-const gsettingsPath = "/usr/bin/gsettings"
+const gsettingsPath = "gsettings"
 
 const allowedCharacters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_-."
 
diff --git a/pkg/osquery/tables/ioreg/ioreg.go b/pkg/osquery/tables/ioreg/ioreg.go
index 5bc91fc..99c3bd4 100644
--- a/pkg/osquery/tables/ioreg/ioreg.go
+++ b/pkg/osquery/tables/ioreg/ioreg.go
@@ -23,7 +23,7 @@ import (
 	"github.com/osquery/osquery-go/plugin/table"
 )
 
-const ioregPath = "/usr/sbin/ioreg"
+const ioregPath = "ioreg"
 
 const allowedCharacters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
 
diff --git a/pkg/osquery/tables/xrdb/xrdb.go b/pkg/osquery/tables/xrdb/xrdb.go
index 8316ffa..cc4f832 100644
--- a/pkg/osquery/tables/xrdb/xrdb.go
+++ b/pkg/osquery/tables/xrdb/xrdb.go
@@ -26,7 +26,7 @@ import (
 	"github.com/pkg/errors"
 )
 
-var xrdbPath = "/usr/bin/xrdb"
+var xrdbPath = "xrdb"
 
 const allowedUsernameCharacters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_-."
 const allowedDisplayCharacters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789:."
diff --git a/pkg/osquery/tables/zfs/tables.go b/pkg/osquery/tables/zfs/tables.go
index 79ef875..694ccba 100644
--- a/pkg/osquery/tables/zfs/tables.go
+++ b/pkg/osquery/tables/zfs/tables.go
@@ -16,8 +16,8 @@ import (
 )
 
 const (
-	zfsPath   = "/usr/sbin/zfs"
-	zpoolPath = "/usr/sbin/zpool"
+	zfsPath   = "zfs"
+	zpoolPath = "zpool"
 )
 
 const allowedCharacters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_-.@/"
